src/content/docs/journey-app-slow/create-benchmarks.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Create performance benchmarks with service levels'
```

Comment:
This doc has solid tasks thru the product and good conceptual info in the Why? but I think it could benefit from defining the why of the specific tasks, rather than just the why? of service levels in the general. 

What might cohere these two sides together is painting a very clear picture of what outcome they achieved by doing all this beyond the thing they created.